shell/imports/server/acme.js

```diff
@@ -24,7 +24,7 @@ import { pki, asn1 } from "node-forge";
 import { SandstormDb } from "/imports/sandstorm-db/db.js";
 import { globalDb } from "/imports/db-deprecated.js";
 import URL from "url";
-import { getSandcatsAcmeOptions, getSandcatsName } from "/imports/server/sandcats.js";
```

Comment:
It looks like this is also the only module that was using `getSandcatsName`, so I think you can delete the actual definition as well.